blockdevice/options.go

```diff
@@ -35,10 +36,18 @@ func WithMode(value int) Option {
 	}
 }
 
+// WithoutGPT opens blockdevice ignoring GPT.
+func WithoutGPT(value bool) Option {
+	return func(args *Options) {
+		args.IgnoreGPT = value
+	}
+}
+
 // NewDefaultOptions initializes a Options struct with default values.
 func NewDefaultOptions(setters ...Option) *Options {
 	opts := &Options{
 		CreateGPT: false,
+		IgnoreGPT: false,
```

Comment:
I don't see so far why do we need this flag. 

It looks like `IgnoreGPT` is something we always had before - if the GPT partition is missing, `Open` still worked, but partition table was nil.

Should we just treat GPT header failures same way as missing PMBR?